be/src/cloud/cloud_compaction_action.cpp

```diff
@@ -196,10 +196,16 @@ Status CloudCompactionAction::_handle_run_status_compaction(HttpRequest* req,
         std::string compaction_type;
         bool run_status = false;
 
+        CloudTabletSPtr cloud_tablet = DORIS_TRY(_engine.tablet_mgr().get_tablet(tablet_id));
```

Comment:
ditto

Reply:
this code seems hard to use defer samentic.